lib/python/qmk/info.py

```diff
@@ -938,7 +940,7 @@ def keymap_json(keyboard, keymap):
     keymap_file = keymap_folder / 'keymap.json'
 
     # Build the info.json file
-    kb_info_json = info_json(keyboard)
+    kb_info_json = info_json(keyboard, skip_validation=True)  # No need to run validation here, we'll do it after load
```

Comment:
Is it OK to ignore invalid entries in the keyboard level `info.json`, if they end up being overridden by valid values at the keymap level?

Reply:
I'd say so, because lint and the like will flag the keyboard-level stuff anyway. We're only really concerned about the end result -- if someone misconfigures a keyboard in their own repo and then submits it for review to upstream QMK, we should notice. The lint workflow disregards keymaps in one of its checks.